src/__stories__/Utilities/Badge.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+
+import { withKnobs } from '@storybook/addon-knobs';
+import { select } from '@storybook/addon-knobs/react';
+import { storiesOf } from '@storybook/react';
+
+import { Badge, BadgeColor, BadgeVariant } from '../../Badge';
+
+export const badgeKnobs = (tabName = 'Badge'): any => {
+  return {
+    color: select('color', [BadgeColor.Error, BadgeColor.Warning, BadgeColor.Default], BadgeColor.Default, tabName),
```

Comment:
Instead of manually listing them out, you could use `Object.values(BadgeColor)`.

Reply:
👍